README.md

```diff
@@ -4,6 +4,9 @@
 
 This is a basic example that helps to scan uploaded resources for malwares with clamd library.
 
+Note: This interfaces via the iUploader system and needs to be first but does not return an object allowing 
```

Comment:
May be worth mentioning that to make it work, this plugin must come earlier in the configured plugin order than any other `IUploader` implementations.

Reply:
do you have wording you want for this, i kinda said it but needs to be clearer